### PR TITLE
Add OpenLMLab/MOSS

### PR DESCRIPTION
**Pull request by gptbert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,3 +243,11 @@ Related links:
 - [A bit translated readme to English](https://github.com/nichtdax/awesome-totally-open-chatgpt/issues/18#issuecomment-1492826662)
 
 Tags: Full
+
+
+## [OpenLMLab/MOSS]([https://github.com/clue-ai/ChatYuan](https://github.com/OpenLMLab/MOSS))
```
Comment (dinhanhx) [inline README.md:248]: This should be `[OpenLMLab/MOSS](https://github.com/OpenLMLab/MOSS)`

File: README.md [hunk 2]
```diff
@@ -242,3 +243,11 @@ Related links:
...
+
+MOSS: An open-source tool-augmented conversational language model from Fudan University (The repos are mostly in Chinese)
+
```
Comment (dinhanhx) [inline README.md:251]: Please add 
```
Related links:
- [English Readme](https://github.com/OpenLMLab/MOSS/blob/main/README_en.md)
```

**Review by dinhanhx — CHANGES_REQUESTED** [review]
Please add an entry in TOC section too

**Comment by nichtdax** [comment]
@gptbert Can you explain how is this project a derivative of LLaMa?

Quoted from [their english readme](https://github.com/OpenLMLab/MOSS/blob/main/README_en.md#:~:text=GPTQ/GPTQ%2Dfor%2DLLaMa%3A%20Quantization%20and%20inference%20backend.), 
> [GPTQ](https://github.com/IST-DASLab/gptq)/[GPTQ-for-LLaMa](https://github.com/qwopqwop200/GPTQ-for-LLaMa): Quantization and inference backend.
They only use the quantization and inference code from another project that support LLaMa

**Pull request merged**